sites/public/src/components/listings/search/MultiSelectGroup.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react"
+import { FormOption } from "./ListingsSearchForm"
+
+type MultiSelectGroupProps = {
+  name: string
+  inputs: FormOption[]
+  values?: string[]
+  onChange: (name: string, value: string[]) => void
+}
+
+/**
+ * Creates a group of checkboxes and sets an array of selected values onChange()
+ *
+ * @param props
+ * @returns
+ */
+export function MultiSelectGroup(props: MultiSelectGroupProps) {
```

Comment:
This is used for counties.  It prints them out as a single column, but it shouldn't be hard to split that up now if needed prior to subsequent UI updates

Reply:
This one definitely looks like it could just use the FieldGroup component with checkboxes https://storybook.bloom.exygy.dev/?path=/story/forms-fieldgroup--field-group-default. If you looked at it, is there a reason why you decided not to use it? And if you didn't, does it seem like a good match?

Reply:
The reason why I didn't use any of the UIC components when building this originally was because I didn't know where any of them were or what options were available until after it was already built.  Replacing these custom form components with UIC components is definitely expected, but the specific issue being resolved here is getting a functional search form in place with the understanding that it would be reskinned later.